CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h

```diff
@@ -120,8 +124,8 @@ std::unique_ptr<TData> L1ConfigOnlineProdBase<TRcd, TData>::produce(const TRcd&
     if (m_copyFromCondDB) {
       // Get L1TriggerKeyList from EventSetup
```

Comment:
btw this comment here is not actually in line with the code below, right? You are not getting the L1TriggerKeyList from the ES directly but from the Record